nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/items/connection/create-connection/create-connection.component.scss

```diff
@@ -20,16 +20,6 @@
 .create-connection-form {
     @include mat.button-density(-1);
 
-    .mdc-dialog__content {
-        padding: 0 16px;
```

Comment:
I see this rule is removed for all the tabbed dialog. Do we need to reapply it in `_app.scss`? If not, then can we remove this rule from the other dialogs? There are only four of them:

![Screenshot 2024-05-09 at 9 50 05 AM](https://github.com/apache/nifi/assets/6797571/10528144-bb60-419b-a1de-f1447f781ca5)